.circleci/config.yml

```diff
@@ -23,7 +23,6 @@ jobs:
           paths:
             - "venv"
       - run:
-          shell: /bin/bash
```

Comment:
same here, while circle-ci says that bash is the default environment, if it is installed, and the shebang also does this; why not be very explicit that this is a hard requirement for the test scripts?

Reply:
because of the shebang and being run with ./ it doesn't matter what this shell is

Reply:
while you are fully right; again, even circle-ci says 'if bash is available, it is the default' Why not make it explicit?

Reply:
because, again, this parameter has no bearing on the shell used for the test script because it is executed with ./ therefor using the interpreter specified by the shebang